shimmy/openspiel_compatibility.py

```diff
@@ -105,14 +110,20 @@ def action_space(self, agent: AgentID):
             )
 
     def render(self):
-        """render."""
-        raise NotImplementedError("No render available for openspiel.")
+        """render.
+
+        Print the current game state.
+        """
+        print(self.game_state)
```

Comment:
This is kinda weird since some openspiel environments actually have a render function, but I guess c'est la vie.

Reply:
I wasn’t actually aware of that, the current code just ignores rendering and says it’s not implemented but I could check if the env has a render function and if not just print the state.